configuration/configuration.go

```diff
@@ -145,6 +145,18 @@ type Configuration struct {
 	Health Health `yaml:"health,omitempty"`
 
 	Proxy Proxy `yaml:"proxy,omitempty"`
+
+	// Schema1Compability configures how schema1 manifests will be handled
+	Schema1Compatibility struct {
+		// TrustKey is the signing key to use for adding the signature to
+		// schema1 manifests.
+		TrustKey string `yaml:"signingkeyfile,omitempty"`
+
+		// DisableSignatureStore will cause all signatures attached to schema1 manifests
+		// to be ignored. Signatures will be generated on all schema1 manifest requests
+		// rather than only requests which converted schema2 to schema1.
+		DisableSignatureStore bool `yaml:"disablesignatures,omitempty"`
+	} `yaml:"schema1compatibility,omitempty"`
```

Comment:
The naming of this section almost seems unnecessary. Does not belong at the top level but not sure where else this could go or if this could be named differently.